Configuration/StandardSequences/python/DigiToRaw_cff.py

```diff
@@ -45,10 +45,11 @@
 
 # GEM settings
 _gem_DigiToRaw = DigiToRaw.copy()
-_gem_DigiToRaw.insert(-1,gemPacker)
+_gem_DigiToRaw.insert(-2,gemPacker)
 
 from Configuration.Eras.Modifier_run2_GEM_2017_cff import run2_GEM_2017
 run2_GEM_2017.toReplaceWith(DigiToRaw, _gem_DigiToRaw)
+run2_GEM_2017.toModify(gemPacker, useDBEMap = cms.bool(True))
```

Comment:
This customization would be better done in the file where `gemPacker` is defined.

Reply:
these are only defined in fillDescriptions in the GEMDigiToRawModule.cc producer

Reply:
In that case it would be better to change the module label here
https://github.com/cms-sw/cmssw/blob/a124be10d5d3f034ab5fbae4fc0b38364abfb82c/EventFilter/GEMRawToDigi/plugins/GEMDigiToRawModule.cc#L31-L38
to e.g. `gemPackerDefault` and add `gemPacker_cfi.py` with the content
```python
import FWCore.ParameterSet.Config as cms
from EventFilter.GEMRawToDigi.gemPackerDefault_cfi import gemPackerDefault as _gemPackerDefault
gemPacker = _gemPackerDefault.clone()

from Configuration.Eras.Modifier_run2_GEM_2017_cff import run2_GEM_2017
run2_GEM_2017.toModify(gemPacker, useDBEMap = True)
```
It is important to apply all customizations in the same file where the module is defined in order to guarantee that the `gemPacker` will be exactly the same independently of how one imports/loads configuration fragments. For an example elsewhere see e.g.
https://github.com/cms-sw/cmssw/blob/master/SimTracker/TrackerHitAssociation/plugins/ClusterTPAssociationProducer.cc#L73
https://github.com/cms-sw/cmssw/blob/master/SimTracker/TrackerHitAssociation/python/tpClusterProducer_cfi.py

(also note that I dropped the explicit `cms.bool` type from the right-hand side, in which case framework will check for typos in the parameter name, i.e. it would give an error if the parameter `useDBEMap` would not already exist)

Reply:
yep, I will update this